src/javascript/javascript-parser.ts

```diff
@@ -27,6 +27,16 @@ declare class SyntaxError {
   column: number;
 }
 
+export interface Comment {
```

Comment:
I had that at first, but it prevented me from casting `Comment` into `Node` when using it with `document.sourceRangeForNode` which I thought was easier than to changing the function signature: https://github.com/MaKleSoft/polymer-analyzer/blob/scan-pseudo-elements-js/src/polymer/pseudo-element-scanner.ts#L75